### PR TITLE
Release 3.2.0

### PR DESCRIPTION
**Pull request by chenliangde**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 All notable changes to this project will be documented in this file.
 This project adheres to Semantic Versioning(http://semver.org/).
 
+## 3.2.0
+- Support add to cart popup recommendations
```
Comment (hannunosto) [inline CHANGELOG.md]: Add support for add to cart popup recommendations

File: classes/services/NostoCartService.php [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+    const QUANTITY = 'quantity';
+
+    const OPERATOR = 'operator';
+
+    const OPERATOR_UP = 'up';
+
+    const ID_PRODUCT_ATTRIBUTE = 'id_product_attribute';
+
+    public function cartUpdated($params)
```
Comment (hannunosto) [inline classes/services/NostoCartService.php]: Doc block missing

File: classes/services/NostoCartService.php [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+                }
+                $service = new NostoSDKCartOperation($account);
+                $service->updateCart($cartUpdate, $nostoCustomerId, $account->getName());
+            }
+        } catch (\Exception $e) {
+            NostoHelperLogger::error($e);
+        }
+    }
+
+    public function cartItemQuantityChanged($params)
```
Comment (hannunosto) [inline classes/services/NostoCartService.php]: Doc block missing

File: CHANGELOG.md (hunk 1 above)
Comment (chenliangde) [reply CHANGELOG.md]: Updated

File: composer.json [hunk 4]
```diff
@@ -2,7 +2,7 @@
   "name": "nosto/nostotagging",
   "description": "Nosto Module for Prestashop",
   "require": {
-    "nosto/php-sdk": "3.1.5"
+    "nosto/php-sdk": "3.1.7"
```
Comment (hannunosto) [inline composer.json]: Can we bump to 3.2.0?

**Pull request merged**